src/sim-host/ui/sim-host.html

```diff
@@ -27,8 +27,8 @@
     </template>
 
     <template id="cordova-dialog-template">
-        <section class="cordova-panel-inner">
-            <section class="cordova-header"><span></span><span class="cordova-close-icon"><svg focusable="false" viewBox="0 0 14 14">
+        <section class="cordova-panel-inner" tabindex="0">
```

Comment:
With this change, what happens when the dialog displays? We probably want the dialog title and contents read out. Currently there is logic to set the focus to the first focusable element in the dialog rather than to the dialog itself. Does that logic now set the focus to the dialog, or should we change it to do so?